test/downstream-project-cmake-test/CMakeLists.txt

```
@@ -13,5 +13,35 @@ else()
   message(STATUS "The current exported EDM4HEP version is " ${EDM4HEP_VERSION})
 endif()
 
+# Use the EDM4HEP_DATA_DIR to derive extended data model
+message(STATUS "${EDM4HEP_DATA_DIR}")
+if ("${EDM4HEP_DATA_DIR}" STREQUAL "")
+  message(WARNING "Unable to extend data model since no installed yaml file found.")
```

```suggestion
  message(FATAL_ERROR "Unable to extend data model since no installed yaml file found.")
```

For the tests this should fail I think to make sure that we can actually build against this.

---

The main reason for not having it be FATAL_ERROR was that I wasn't sure of the timeline for the required changes to podio and EDM4hep. It seems that's moving forward quickly though so we can expect this to pass tests again soon.